tests/tooltips/colorPicker_test.js

```diff
@@ -1,55 +1,81 @@
+import {getLine, getMockedTooltip, getTooltip, testMockedTooltipDetection, testReplace, typeLine} from "./shared.js";
+import ColorPicker from "../../js/ui/tooltips/color-picker.js";
+
 describe("colorPicker - detection", function() {
-    var mockedColorPicker = getMockedTooltip(tooltipClasses.colorPicker, ["detector", "initialize"]);
+
+    let sandbox;
```

Comment:
I changed the tooltip tests to use sinon sandbox. It seemed safer now that tests are being bundled together, and its generally a good practice so that you can restore.

Reply:
How does sinon's `sandbox` differ from what we were doing before?

Reply:
Also, these sandbox changes probably would made a good PR on their own.

Reply:
Good call, these changes can go straight to master, I've sent a PR.